scripts/zones/Upper_Jeuno/npcs/Guslam.lua

```diff
@@ -46,63 +46,63 @@ function onTrigger(player,npc)
 			player:startEvent(0x009b); -- Start Quest for THF
 		elseif(player:getMainJob() == 7 and 
-			   player:getQuestStatus(SANDORIA,A_BOY_S_DREAM) == QUEST_COMPLETED and 
+			   player:getQuestStatus(SANDORIA,A_BOY_S_DREAM) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_STALWART_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for PLD
 		elseif(player:getMainJob() == 8 and 
-			   player:getQuestStatus(BASTOK,DARK_PUPPET) == QUEST_COMPLETED and 
+			   player:getQuestStatus(BASTOK,DARK_PUPPET) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_SHADOWY_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for DRK
 		elseif(player:getMainJob() == 9 and 
-			   player:getQuestStatus(JEUNO,SCATTERED_INTO_SHADOW) == QUEST_COMPLETED and 
+			   player:getQuestStatus(JEUNO,SCATTERED_INTO_SHADOW) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_WILD_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for BST
 		elseif(player:getMainJob() == 10 and 
-			   player:getQuestStatus(JEUNO,THE_REQUIEM) == QUEST_COMPLETED and 
+			   player:getQuestStatus(JEUNO,THE_REQUIEM) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_HARMONIOUS_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for BRD
 		elseif(player:getMainJob() == 11 and 
-			   player:getQuestStatus(WINDURST,FIRE_AND_BRIMSTONE) == QUEST_COMPLETED and 
+			   player:getQuestStatus(WINDURST,FIRE_AND_BRIMSTONE) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_CHASING_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for RNG
 		elseif(player:getMainJob() == 12 and 
-			   player:getQuestStatus(OUTLANDS,YOMI_OKURI) == QUEST_COMPLETED and 
+			   player:getQuestStatus(OUTLANDS,YOMI_OKURI) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_LOYAL_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for SAM
 		elseif(player:getMainJob() == 13 and 
-			   player:getQuestStatus(OUTLANDS,I_LL_TAKE_THE_BIG_BOX) == QUEST_COMPLETED and 
+			   player:getQuestStatus(OUTLANDS,I_LL_TAKE_THE_BIG_BOX) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_LURKING_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for NIN
 		elseif(player:getMainJob() == 14 and 
-			   player:getQuestStatus(SANDORIA,CHASING_QUOTAS) == QUEST_COMPLETED and 
+			   player:getQuestStatus(SANDORIA,CHASING_QUOTAS) ~= QUEST_AVAILABLE and 
 			   player:getQuestStatus(JEUNO,BORGHERTZ_S_DRAGON_HANDS) == QUEST_AVAILABLE) then 
 			player:startEvent(0x009b); -- Start Quest for DRG
 		elseif(player:getMainJob() == 15 and 
-			   player:getQuestStatus(WINDURST,CLASS_REUNION) == QUEST_COMPLETED and 
+			   player:getQuestStatus(WINDURST,CLASS_REUNION) ~= QUEST_AVAILABLE and 

```

Comment:
#1826 "BLM AF3 quest could be started when AF2 quest is accepted. This is not right - AF2 needs to be completed before AF3 can begin, so I added a check for that as well."